src/main/java/com/unb/tracker/validator/UserValidator.java

```diff
@@ -0,0 +1,40 @@
+package com.unb.tracker.validator;
+
+import com.unb.tracker.model.User;
+import com.unb.tracker.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+@Component
+public class UserValidator implements Validator {
+	@Autowired
+	private UserService userService;
+
+	@Override
+	public boolean supports(Class<?> aClass) {
```

Comment:
This doesn't even seem to be called explicitly, does springboot call this?

Reply:
Actually not sure about this one. This was in a tutorial we found, so I just left it in. Looked important ;)